helpers/validators.js

```diff
@@ -27,8 +31,20 @@ const emailValidator = {
   }
 }
 
+const phoneOnKeyPress = (e) => {
+  const charCode = e.which ? e.which : e.keyCode;
+
+  if (charCode == 43){
```

Comment:
Using ASCII table numbers makes this harder to understand.
Use `charCodeAt` to make comparisons to improve readability:
```suggestion
  if (charCode == '+'.charCodeAt()){
```

Reply:
Applied changes